### PR TITLE
Calm down rectangle selection

### PR DESCRIPTION
**Pull request by Don-Vito**
It seems that our Rectangle Selection is not aware of the fact
that brain and fingers are not connected.
So if you click and move a mouse a tiny bit, it already gets triggered.
This is something not really pleasant, as it might cancel shift selection
of the grid, or to unregister `AllView_ItemPressed`, actually preventing
activation in one-click mode.

This PR verifies that the mouse has moved sufficiently to indicate
rectangle selection (the value is hard-coded right now - please
let me know if you have better ideas).

BTW, probably the better solution is radius, rather than checking
the dimensions, but I doubt the ROI :smile:

### CODE REVIEW

File: Files/UserControls/Selection/RectangleSelection_DataGrid.cs [hunk 1]
```diff
@@ -37,17 +37,28 @@ public RectangleSelection_DataGrid(DataGrid uiElement, Rectangle selectionRectan
 
         private void RectangleSelection_PointerMoved(object sender, PointerRoutedEventArgs e)
         {
+            if (scrollBar == null)
```
Comment (d2dyno1) [inline Files/UserControls/Selection/RectangleSelection_DataGrid.cs:40]: More fancier way 😜 `if (scrollBar is not ScrollBar)`

Comment (Don-Vito) [reply Files/UserControls/Selection/RectangleSelection_DataGrid.cs:40]: This definitely looks more philosophical this way :smile:
I preserved the original semantics (that were in line 48)

Comment (gave92) [reply Files/UserControls/Selection/RectangleSelection_DataGrid.cs:40]: @d2dyno1 how could you! you are missing the brackets, please follow Files code-style!! 😂

Comment (d2dyno1) [reply Files/UserControls/Selection/RectangleSelection_DataGrid.cs:40]: NO_I_WILL_NOT

**Comment by gave92** [comment]
Also here this issue is back #2749?

**Comment by Don-Vito** [comment]
> 
> 
> Also here this issue is back #2749?

@gave92 - doesn't happen to me. Also see no reason why it would. 🤔

**Comment by Don-Vito** [comment]
@gave92 - fixed.. by rebasing to master :smile:

**Comment by gave92** [comment]
@Don-Vito uops xD I just assumed it already was. Thanks

**Review by gave92 — APPROVED** [review]
@Don-Vito great work 👍

**Comment by Thomas1664** [comment]
> BTW, probably the better solution is radius, rather than checking
> the dimensions, but I doubt the ROI 😄

@Don-Vito If you want radius you can use the vector-distance-formula like so:
```c#
private double CalcVectorDistance(int x1, int y1, int x2, int y2)
{
    return Math.Sqrt(Math.Pow((x2 - x1), 2) + Math.Pow((y2 - y1), 2));
}
```

**Pull request merged**